sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/ConfidentialLedgerClientTestBase.java

```diff
@@ -41,89 +40,83 @@ class ConfidentialLedgerClientTestBase extends TestBase {
 
     @Override
     protected void beforeTest() {
+
         try {
             ConfidentialLedgerCertificateClientBuilder confidentialLedgerCertificateClientBuilder = new ConfidentialLedgerCertificateClientBuilder()
                 .certificateEndpoint("https://identity.confidential-ledger.core.azure.com")
-                .httpClient(HttpClient.createDefault())
                 .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC));
+
             if (getTestMode() == TestMode.PLAYBACK) {
                 confidentialLedgerCertificateClientBuilder
-                        .httpClient(interceptorManager.getPlaybackClient())
-                        .credential(request -> Mono.just(new AccessToken("this_is_a_token", OffsetDateTime.MAX)));
+                    .httpClient(interceptorManager.getPlaybackClient())
+                    .credential(request -> Mono.just(new AccessToken("this_is_a_token", OffsetDateTime.MAX)));
             } else if (getTestMode() == TestMode.RECORD) {
                 confidentialLedgerCertificateClientBuilder
-                        .addPolicy(interceptorManager.getRecordPolicy())
-                        .credential(new DefaultAzureCredentialBuilder().build());
+                    .addPolicy(interceptorManager.getRecordPolicy())
+                    .credential(new DefaultAzureCredentialBuilder().build());
             } else if (getTestMode() == TestMode.LIVE) {
                 confidentialLedgerCertificateClientBuilder.credential(new DefaultAzureCredentialBuilder().build());
             }
-            confidentialLedgerCertificateClient = confidentialLedgerCertificateClientBuilder
-                    .buildClient();
-            
-            String ledgerId = Configuration.getGlobalConfiguration().get("LEDGER_NAME", "java-sdk-test");
+
+            confidentialLedgerCertificateClient = confidentialLedgerCertificateClientBuilder.buildClient();
+
+            String ledgerName = Configuration.getGlobalConfiguration().get("LEDGER_NAME", "emily-java-sdk-tests");
```

Comment:
I think we should use a more generic name for the ledger like what you used to have.

Reply:
Done!